ivy_tests/test_docstrings.py

```diff
@@ -131,6 +232,13 @@ def check_docstring_examples_run(
         if ">>> print(" in line:
             is_multiline_executable = False
 
+    # remove "..." for multilines
+    for i, v in enumerate(executable_lines):
+        executable_lines[i] = v.replace("...", "")
+        # backend which doesn't support inplace_update throws an exception
+        executable_lines[i] = v.replace(
```

Comment:
I don't think we should be having a `ensure_in_backend=True` example in the `inplace_updates` function, given that it's an expected inconsistency, or we should be able to skip the docstring tests for `jax` and `tensorflow` if we want to keep that example.

Reply:
this fails but when `ensure_in_backend` not explicitly set to `True` it is fine.(since default is `False`. Should I add it to skip list?
```py
ivy.set_backend("jax")
x = ivy.array([1, 2, 3])
y = ivy.array([0])
x.inplace_update(y, ensure_in_backend=True)
print(x)
```

Reply:
I think given that this inconsistency is something we've already communicated in the [docs](https://unify.ai/docs/ivy/overview/deep_dive/inplace_updates.html#inplace-updates) I think we could probably avoid using examples with `ensure_in_backend=True` in the docstrings 😄